<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigFileDialog</class>
 <widget class="QDialog" name="ConfigFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config File Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_content">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration File Content:</string>
   </property>
   <property name="buddy">
    <cstring>txt_configFile</cstring>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_configFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>661</width>
     <height>611</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_aliases">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>210</y>
     <width>511</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>511</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="lbl_aliases">
      <property name="text">
       <string>Available Aliases:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="buddy">
       <cstring>txt_aliases</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbl_selectType">
      <property name="text">
       <string>Select Rule Type:</string>
      </property>
      <property name="buddy">
       <cstring>cmb_ruleType</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn_generate">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
      <property name="icon">
       <iconset resource="../res.qrc">
        <normaloff>:/resources/res/icons/GenerateRule.png</normaloff>:/resources/res/icons/GenerateRule.png</iconset>
      </property>
      <property name="shortcut">
       <string>Ctrl+G</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cmb_ruleType">
      <item>
       <property name="text">
        <string>1) SINGLE RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2) SINGLEWITHSCRIPT RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3) SINGLEWITHSUPPRESS RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4) PAIR RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5) PAIRWITHWINDOW RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6) SINGLEWITHTHRESHOLD RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7) SINGLEWITH2THRESHOLDS RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8) EVENTGROUP RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9) SUPPRESS RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10) CALENDAR RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11) JUMP RULE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12) OPTIONS RULE</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_addRule">
      <property name="text">
       <string>Add Rule:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
      <property name="buddy">
       <cstring>cmb_ruleType</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_saveAndExit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>SAVE ALL + CLOSE</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/resources/res/icons/Save.png</normaloff>:/resources/res/icons/Save.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1181</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_config">
      <property name="text">
       <string>Configuration File Name:</string>
      </property>
      <property name="buddy">
       <cstring>txt_confName</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="txt_confName">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_configFile</tabstop>
  <tabstop>cmb_ruleType</tabstop>
  <tabstop>btn_generate</tabstop>
  <tabstop>txt_aliases</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
